events/ecr_image_action.go

```diff
@@ -0,0 +1,21 @@
+package events
+
+type ECRImageActionEvent struct {
+	Version    string                        `json:"version"`
+	ID         string                        `json:"id"`
+	DetailType string                        `json:"detail-type"`
+	Source     string                        `json:"source"`
+	Account    string                        `json:"account"`
+	Time       string                        `json:"time"`
```

Comment:
I think this can be `time.Time`. There's precedent in other eventbridge modeled events we have in this project. ex: https://github.com/aws/aws-lambda-go/blob/main/events/codedeploy.go#L23-L80